beta/src/pages/learn/conditional-rendering.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: Conditional Rendering
+title: कंडिशनल  रेण्डरिंग
```

Comment:
`रेण्डरिंग` => `रेंडरिंग`

I think this would be the right spelling. we have been using this everywhere

Reply:
👍